src/fsharp/CheckFormatStrings.fsi

```diff
@@ -10,19 +10,26 @@ module internal FSharp.Compiler.CheckFormatStrings
 open FSharp.Compiler.NameResolution
 open FSharp.Compiler.TcGlobals
 open FSharp.Compiler.Text
-open FSharp.Compiler.TypedTree 
+open FSharp.Compiler.TypedTree
 
 val ParseFormatString:
-    m: range
-    -> fragmentRanges: range list
-    -> g: TcGlobals
-    -> isInterpolated: bool
-    -> isFormattableString: bool
-    -> formatStringCheckContext: FormatStringCheckContext option
-    -> fmt: string
-    -> printerArgTy: TType
-    -> printerResidueTy: TType
-    -> printerResultTy: TType
-        -> TType list * TType * TType * TType[] * (range * int) list * string
+    m: range ->
```

Comment:
Interestingly, here the format is how I like it.

Reply:
Yup, we have the `->` at the end OK.  Full context is linked in https://github.com/fsprojects/fantomas/issues/1955.  THe problem is in mixed curried/tupled signatures but we rarely use those